### ISSUE TITLE
Support for non-standard `GP1`/`GRP1` frames.

### ISSUE THREAD

**Issue by Cantello** [squell/id3#32]
I am trying to copy one tag from field "TXXX:grouping" to field "GRP1". Should be simple in theory but I somehow cannot grasp the concept. I start with `id3 -2 -wTXXX:grouping "%{GRP1}" mp3file.mp3` (similar to the below example) but this does not write the GRP1 field to the TXXX field. If I just add text instead of the %{GRP1}, it works.

Any ideas? It almost seems as if GRP1 is not in the list of compatible frames (as per compatability section), so maybe I just use another TXXX frame to copy to the desired frame...

![image](https://github.com/user-attachments/assets/5960f4bf-d5fb-4bfd-be5c-b8412faf68d4)

**Comment by Cantello** [comment]
I now simply used a workaround where I first write the tag to TXXX:series and then to TXXX:grouping. Works well so far.

**Comment by squell** [comment]
Yes, the `GRP1` frame, is not defined in ID3v2.3 and so not supported. I see that this apparently is an instance where iTunes just defines a custom tag: https://github.com/id3/ID3v2.4/issues/18 without regard for what some person on the internet wrote in the ID3v2 spec 20 years ago. Fair enough. :-)

I recall back in the day iTunes also stubbornly being the only program that would write ID3v2.2 tags.

Also see: https://discussions.apple.com/thread/7900430?sortBy=rank

I'll re-open this issue as a "add reading support for GRP1" (I probably don't want to encourage people to *write* GRP1/GP1 frames). Although it would be nice to have an example of a ID3v2 tag that has one so I can see what the structure is (otherwise I would have to guess). Maybe you can send one?
